src/style.css

```diff
@@ -0,0 +1,99 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 600;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+form {
+  width: 100%;
+  height: 70px;
+}
+
+.form-small {
+  width: 100%;
+  height: 42px;
+}
+
+.label {
+  width: 100%;
+  display: block;
+}
+
+.input {
+  height: 100%;
+  width: 100%;
+  font-family: inherit;
+  font-weight: 300;
+  border-radius: 4px;
+  color: rgb(61, 78, 97);
+  border: 1px solid rgb(225, 231, 237);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  background-image: url(../src/images/Search.svg);
+  background-repeat: no-repeat;
+  outline: none;
+}
+
+.input::placeholder {
+  color: inherit;
+  font-weight: 300;
+}
+
+input {
```

Comment:
Not fixed
![image](https://user-images.githubusercontent.com/54936044/231732155-5cf630f0-a3f0-40cc-be0b-0be5f930e8c9.png)